runtime/qsym_backend/Runtime.cpp

```diff
@@ -449,7 +469,7 @@ bool _sym_feasible(SymExpr expr) {
 //
 
 void _sym_collect_garbage() {
-  if (allocatedExpressions.size() < g_config.garbageCollectionThreshold)
+//  if (allocatedExpressions.size() < g_config.garbageCollectionThreshold)
```

Comment:
Why you left that comment?

Reply:
The tracer identifies each expression with a unique ID, that is used in the JSON file. Currently, I used the memory address of the expression as ID and thus to avoid ID collision we need to disable the garbage collection.

As discussed above, I will put the code of this PR inside `#ifdef`s such that garbage collection is disabled only when performing tracing.

Reply:
So maybe changing way of creating IDs will be a good idea? I am afraid that after turning off garbage collection the memory usage can explode.